src/manifests_base.py

```diff
@@ -251,3 +248,94 @@ def from_space_separated(cls, tolerations: str) -> List["CephToleration"]:
             return [cls._from_string(toleration) for toleration in tolerations.split()]
         except ValueError as e:
             raise ValueError(f"Invalid tolerations: {e}") from e
+
+
+class ProvisionerAdjustments(Patch):
```

Comment:
This class was yanked from each of RBDManifest and CephFSManifest and consolidated here.  It does the same jobs it did before, now it's also responsible for adjusting the csidriver name where it can.